### PR TITLE
Store and Components

### PR DESCRIPTION
**Pull request by AymenBida**
In this milestone I did: 

- Create different components for the different parts of the app (book, booksList, booksForm)
- Create a Redux store and connected it to the booksList component

### CODE REVIEW

**Review by arslanbisharat — COMMENTED** [review]
I cannot review the project https://github.com/AymenBida/magic-books/pull/2/ at this moment as I have urgent work to do. Please request another review.

File: src/containers/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import BooksList from '../components/BooksList';
```
Comment (afanimartin) [inline src/containers/App.js]: - Move this component to `src/components`

File: src/components/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import getBooks from '../redux/selectors';
+import Book from './Book';
+import { CATEGORIES } from './BooksForm';
+
```
Comment (afanimartin) [inline src/components/BooksList.js]: - Move this component to `src/containers` where it rightly belongs

File: src/components/BooksForm.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+export const CATEGORIES = [
+  { name: 'Action', id: 1 },
+  { name: 'Biography', id: 2 },
+  { name: 'History', id: 3 },
+  { name: 'Horror', id: 4 },
+  { name: 'Kids', id: 5 },
+  { name: 'Learning', id: 6 },
+  { name: 'Sci-Fi', id: 7 },
+];
```
Comment (afanimartin) [inline src/components/BooksForm.js]: - Move this to `src/containers`

File: src/redux/actions/index.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { CREATE_BOOK } from './actionTypes';
+// const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (afanimartin) [inline src/redux/actions/index.js]: - Remove this comment

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this milestone.
You managed to fulfill most requirements for this milestone. Well done!
### Changes 🙅🏿 🙅🏿 🙅🏿 
- Remove comments from the codebase
- Move files to the right folders. Check my comments.

### Martin Afani
#### Slack: whiz25

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @AymenBida**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request closed without merging**